exercises/01_penguin_classification.ipynb

```diff
@@ -197,11 +202,14 @@
     "- Instantiate the penguin dataloader.\n",
     "  - Make sure you supply the correct column titles for the features and the targets.\n",
     "- Iterate over the dataset\n",
-    "    - Hint:\n",
-    "        ```python\n",
-    "        for features, targets in dataset:\n",
-    "            # print the features and targets here\n",
-    "        ```"
+    "  <details>\n",
+    "    <summary>Hint:</summary>\n",
+    "\n",
+    "    ```python\n",
+    "      for features, targets in dataset:\n",
+    "          # print the features and targets here\n",
+    "    ```\n",
+    "  </details>"
```

Comment:
This 'hint' is repeated in the code cell below, so should probably just remove it from here.
```suggestion
```